Panic occurs after about 10 seconds of inactivity after initialization

When I try the setup in the README, panic occurs after about 10 seconds of inactivity after flashDB initialization.

```shell
2022-03-21 0:19:24 ~/p/g/g/a/flashdb (git)-[main]
% go build -o bin/flashdb-server cmd/server/main.go
2022-03-21 0:19:39 ~/p/g/g/a/flashdb (git)-[main]
% ./bin/flashdb-server
2022/03/21 00:19:40 no config set, using the default config.
2022/03/21 00:19:40 FlashDB is running, ready to accept connections.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x10cfa5c]

goroutine 20 [running]:
github.com/arriqaaq/art.(*Node).IsLeaf(...)
        /Users/matsuyoshi/go/pkg/mod/github.com/arriqaaq/art@v0.1.1/art.go:410
github.com/arriqaaq/flashdb.(*strStore).Keys.func1(0x4b11888)
        /Users/matsuyoshi/projects/ghq/github.com/arriqaaq/flashdb/store.go:45 +0x1c
github.com/arriqaaq/art.(*Tree).each(0x1, 0x0, 0xc00021fdf8)
        /Users/matsuyoshi/go/pkg/mod/github.com/arriqaaq/art@v0.1.1/art.go:811 +0x32
github.com/arriqaaq/art.(*Tree).Each(...)
        /Users/matsuyoshi/go/pkg/mod/github.com/arriqaaq/art@v0.1.1/art.go:807
github.com/arriqaaq/flashdb.(*strStore).Keys(0x101000000000060)
        /Users/matsuyoshi/projects/ghq/github.com/arriqaaq/flashdb/store.go:44 +0x50
github.com/arriqaaq/flashdb.(*strStore).evict(0xc00009c000, 0xc0000a2018)
        /Users/matsuyoshi/projects/ghq/github.com/arriqaaq/flashdb/store.go:57 +0x96
github.com/arriqaaq/flashdb.(*sweeper).run(0xc00009c080, 0x0)
        /Users/matsuyoshi/projects/ghq/github.com/arriqaaq/flashdb/evict.go:58 +0x65
created by github.com/arriqaaq/flashdb.New
        /Users/matsuyoshi/projects/ghq/github.com/arriqaaq/flashdb/flashdb.go:62 +0x857
```

The cause is `tree.Each()` is called for `art.Tree` (nil) in `strStore`.

--- a/art.go
+++ b/art.go
@@ -804,7 +804,9 @@ func (t *Tree) delete(currentRef **Node, key []byte, depth int) bool {
 }
 
 func (t *Tree) Each(callback Callback) {
-	t.each(t.root, callback)
+	if t.Size() > 0 {
+		t.each(t.root, callback)
+	}
 }
 
 func (t *Tree) each(current *Node, callback Callback) {

--- a/art_test.go
+++ b/art_test.go
@@ -877,6 +877,13 @@ func TestWordsWithPrefix(t *testing.T) {
 	}
 }
 
+func TestEachEmptyTree(t *testing.T) {
+	tree := NewTree()
+	tree.Each(func(n *Node) {
+		// nop
+	})
+}
+
 //
 // Benchmarks
 //

